### ISSUE TITLE
Fix broken keyboard navigation in dropdowns

### ISSUE THREAD

**Issue by Pedrinhho** [Pedrinhho/gerenciaBanco#19]
Keyboard navigation does not work properly in dropdown menus. Debug and update logic.
